docker/gateway-mutinynet/docker-compose.yaml

```diff
@@ -45,16 +62,110 @@ services:
       - bitcoind
     restart: always
 
+  thunderhub_config_writer:
+    image: alpine:latest
+    command:
+      - sh
+      - -c
+      - |
+        cat <<EOF > /thconfig/accounts.yaml
+        masterPassword: "thereisnosecondbest"
+        accounts:
+          - name: "main lnd"
+            serverUrl: "https://lnd:10009"
+            lndDir: "/root/.lnd"
+            network: signet
+        EOF
+        tail -f /dev/null
+    volumes:
+      - thunderhub_datadir:/thconfig
+    depends_on:
+      - lnd
+    restart: always
+
+  thunderhub:
```

Comment:
I think we need a `BASE_PATH` config, otherwise you have to just know that you have to visit it at `/thub`: https://docs.thunderhub.io/setup#running-on-different-base-path

<img width="578" alt="Screenshot 2023-07-13 at 3 40 37 PM" src="https://github.com/fedimint/fedimint/assets/649992/5c5d75dc-7f7d-4ad6-8162-1ea7c867a076">

Reply:
good catch. thunderhub is currently broken due to https://github.com/apotdevin/thunderhub/pull/545, so it's hard to test